frontend/components/SingleItemView/SingleItemView.js

```diff
@@ -38,6 +39,7 @@ class SingleItemView extends React.Component {
       stock: this.state.tempStock,
       editMode: false,
     });
+    updateItemStock(this.state) //not sure if this is correct
```

Comment:
You are going to want to .catch this function and possibly display the error message in some way.